### DISCUSSION TITLE
colab

### DISCUSSION THREAD

**Discussion by rafiislambd** [MoonRide303/Fooocus-MRE#82] [General]
please fix colab generated image show slowly .thanks

**Comment by barepixels** [comment]
**Chris Perry, Google Colab lead, confirmed that SD** and RVC **free tier usage is banned**, but pro user can still use it (again)

https://www.reddit.com/r/StableDiffusion/comments/16g2l66/chris_perry_google_colab_lead_confirmed_that_sd/

**Comment by barepixels** [comment]
**Google doesn't want people to use ComfyUI on colab anymore.**

https://github.com/comfyanonymous/ComfyUI/commit/10de64af7f1ea22e08e39267ade7ef7f8b1607fe

**Comment by barepixels** [comment]
**Is it true that colab is starting to actually stop a1111 colabs from working?**

https://www.reddit.com/r/StableDiffusion/comments/16f3x8g/is_it_true_that_colab_is_starting_to_actually/

**Comment by MoonRide303** [comment]
@rafiislambd I just tried current MRE (v2.0.8) in Colab, and it seemes to be working more or less reasonably (under 1 minute in Speed mode).

**Comment by barepixels** [comment]
**WARNING: Attempting to get around the free-tier Colab ban of SD webui will get your account BLOCKED from Colab usage.**

https://www.reddit.com/r/StableDiffusion/comments/16j16c5/warning_attempting_to_get_around_the_freetier/
